src/platform/forms-system/src/js/web-component-patterns/fullNamePattern.js

```diff
@@ -120,6 +120,9 @@ const firstNameLastNameNoSuffixUI = (formatTitle, uiOptions = {}) => {
       },
     },
   };
+  delete uiSchema.middle;
```

Comment:
it looks like this shouldn't be needed, since there is no middle

Reply:
Removed.  Thanks for catching this!